hack/run-test-e2e.sh

```diff
@@ -56,6 +60,21 @@ fi
 
 
 echo "Running Functional Tests: ${GINKGO_SUITS}"
+
+echo "Deploying sample devices for RTE tests"
+rte/hack/deploy-devices.sh
+if [ $? -ne 0 ]; then
+    echo "Failed to deploy sample device plugin for RTE tests"
+    exit 2
```

Comment:
Why different exit codes? we have the log above for indicating where the flow breaks

Reply:
because this way we have a relatively easy way to handle different failures programmatically. In this case however it is going to ben overridden by another change, so I guess it won't matter much